README_fr_FR.md

```diff
@@ -0,0 +1,94 @@
+<h1 align="center">
+<img src="https://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png" alt="Dogecoin" width="300"/>
+<br/><br/>
+Dogecoin Core [DOGE, Ð]  
+</h1>
+
+<div align="center">
+
+[![DogecoinBadge](https://img.shields.io/badge/Doge-Coin-yellow.svg)](https://dogecoin.com)
+[![MuchWow](https://img.shields.io/badge/Much-Wow-yellow.svg)](https://dogecoin.com)
+
+</div>
+
+Sélectionner la langue: [EN](README.md) | [CN](./README_zh_CN.md) | [PT](./README_pt_BR.md) | [VN](./README_vi_VN.md) | [FA](./README_fa_IR.md) | FR
+
+Dogecoin est une crypto-monnaie communautaire qui s'inspire d'un mème de Shiba Inu. Le logiciel Dogecoin Core permet à quiconque d'exploiter un nœud dans les réseaux de la blockchain Dogecoin et utilise la méthode de hachage Scrypt pour la preuve de travail. Il est adapté de Bitcoin Core et d'autres crypto-monnaies.
+
+Pour plus d'informations sur les frais par défaut utilisés sur le réseau Dogecoin, veuillez consulter la [recommandation sur les frais](doc/fee-recommendation.md).
+
+**Site web:** [dogecoin.com](https://dogecoin.com)
+
+## Utilisation 💻
+
+Pour commencer votre voyage avec Dogecoin Core, consultez le [guide d'installation](INSTALL.md) et le [tutoriel de démarrage](doc/getting-started.md).
+
+L'API JSON-RPC fournie par Dogecoin Core est auto-documentée et peut être parcourue avec `dogecoin-cli help`, tandis que des informations détaillées pour chaque commande peuvent être visualisées avec `dogecoin-cli help <command>`. Vous pouvez également consulter la [documentation Bitcoin Core](https://developer.bitcoin.org/reference/rpc/) - qui implémente un protocole similaire - pour obtenir une version consultable.
+
+### Les ports
+
+Dogecoin Core utilise par défaut le port `22556` pour la communication pair-à-pair qui est nécessaire pour synchroniser la blockchain au "mainnet" et rester informé des nouvelles transactions et blocs. De plus, un port JSONRPC peut être ouvert, dont par défaut le port `22555` est utilisé pour les nœuds du réseau principal. Il est fortement recommandé de ne pas exposer les ports RPC à l'Internet public.
```

Comment:
Peut-être laisser "peer-to-peer" au lieu de "pair-à-pair".

Reply:
Je propose qu'on garde pair-à-pair et qu'on standardise une bonne fois pour toute pour éviter de le changer à nouveau tous les ans.